synapse/cores/common.py

```diff
@@ -213,6 +149,14 @@ def __init__(self, link, **conf):
         self.initTufosBy('inet:cidr', self._tufosByInetCidr)
         self.initTufosBy('dark', self._tufosByDark)
 
+        # Initialize the storage layer
+        self.store = store(link, self)  # type: s_storage.Storage()
```

Comment:
we'll probably want to hand in an already constructed storage instance rather than a ctor since we may want to manipulate the storage layer object before cortex takes it over